benchmarks/example/create.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
Why can't you just export this function?

Reply:
Because that doesn't align with the example use-case I want to portray in `README.md` where only the "implementation" changes and the benchmark stays the same.
If both functions are exported I'd have to choose one in `benchmark.js`.